Formula/spack.rb

```diff
@@ -0,0 +1,21 @@
+class Spack < Formula
+  desc "Package manager that builds multiple versions and configurations of software"
+  homepage "https://spack.io"
+  url "https://github.com/spack/spack/archive/v0.16.0.tar.gz"
+  sha256 "064b2532c70916c7684d4c7c973416ac32dd2ea15f5c392654c75258bfc8c6c2"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/spack/spack.git", branch: "develop"
+
+  bottle :unneeded
+
+  depends_on "python"
```

Comment:
Sorry, missed this one last time:
```suggestion
  depends_on "python@3.9"
```